From 3690121152403dc4fb4fe50bb5837007f7a4de68 Mon Sep 17 00:00:00 2001
From: serge-sans-paille <serge.guelton@telecom-bretagne.eu>
Date: Sat, 29 Nov 2025 17:18:38 +0100
Subject: [PATCH] Update gast and beniget requirements

This mostly brings in better gast support for py3.12+ nodes
---
 requirements.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index c7a25c52a..2fcb002e2 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,5 +1,5 @@
 ply>=3.4
 setuptools
-gast~=0.6.0
+gast~=0.7.0
 numpy
-beniget~=0.4.0
+beniget~=0.5.0
